src/autocapture.js

```diff
@@ -46,7 +47,7 @@ var autocapture = {
             // Only capture attributes we know are safe
             if (isSensitiveElement(elem) && ['name', 'id', 'class'].indexOf(attr.name) === -1) return
 
-            if (!maskInputs && shouldCaptureValue(attr.value)) {
+            if (!maskInputs && shouldCaptureValue(attr.value) && !isAngularContentAttr(attr.name)) {
```

Comment:
Sanity check: is attr.name _always_ a string?

Reply:
Good call - added a type check and test for the case